BlocSpot/app/src/main/java/com/izor066/android/blocspot/BlocSpotApplication.java

```diff
@@ -0,0 +1,33 @@
+package com.izor066.android.blocspot;
+
+import android.app.Application;
+
+import com.izor066.android.blocspot.api.DataSource;
+
+/**
+ * Created by igor on 22/9/15.
+ */
+public class BlocSpotApplication  extends Application {
+
+    public static BlocSpotApplication getSharedInstance() {
+        return sharedInstance;
+    }
+
+    public static DataSource getSharedDataSource() {
+        return BlocSpotApplication.getSharedInstance().getDataSource();
+    }
+
+    private static BlocSpotApplication sharedInstance;
```

Comment:
Fields should be at top of the class, so in general the order within a class is:
1. static fields (class fields)
2. instance fields
3. methods